coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,393 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    Formatting refers to the visual appearance of you source code.
```

Comment:
Remove 'you', everywhere. Incorrect grammar, but also irrelevant who possesses the source code.